fast-list.js

```diff
@@ -106,7 +105,7 @@ FastList.prototype = {
     var itemPerScreen = geo.viewportHeight / geo.itemHeight;
     // Taking into account the will-change budget multiplier from
     // layout/base/nsDisplayList.cpp#1193
-    geo.maxItemCount = Math.floor(itemPerScreen * 2.4);
+    geo.maxItemCount = Math.floor(itemPerScreen * 2.8);
```

Comment:
this change snuck in as part of an earlier commit. but the issue I was trying to fix is actually gaia or gecko related but not from the list.